### PR TITLE
:pencil: Updated todo with new links

### PR DESCRIPTION
**Pull request by SpyHoodle**
- Added new links
- Added new sections
- Made it look cleaner and more organised

### CODE REVIEW

**Comment by SpyHoodle** [comment]
Done!

File: README.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
-# Together
-The website for the Code Together Discord server
-
+# Coded Together
+The [website](https://together.codes) for the code::together [Discord Server](https://discordapp.com/invite/HPK3hB6)<br>
+Published under the [MIT Licence](https://github.com/codedtogether/web/blob/master/LICENSE).
```
Comment (abcdan) [inline README.md]: I don't think this is needed to be added, as it's already in a separate file

File: README.md [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 # Setup
+Want to see what we've done so far? Simply follow the steps.
 1. Go to the `root` folder of the project, so where you are probably right now :)
 2. Run `npm install`
 3. Run `npm run dev & open http://localhost:3000`
 4. Navigate to `localhost:3000` in your browser
 
 
 # Like to contribute?
+Yay! We'd love your help!<br>
```
Comment (abcdan) [inline README.md]: We don't need this line in my opinion

File: README.md [hunk 3]
```diff
@@ -18,5 +20,8 @@ We use `Svelte 3` and `Sapper` to build our website, they are both amazing libra
 Next to that, we use Sveltestrap to easily use Bootstrap components in our website. For the docs, check [this](https://bestguy.github.io/sveltestrap/) page.
 
 
-# Discord
-Join our Discord! Click [here](https://discordapp.com/invite/HPK3hB6)
+# Discord Server
+Join our Discord Server at [discord.gg/HPK3hB6](https://discordapp.com/invite/HPK3hB6)
+
+# Thanks!
+Thanks to our amazing [contributiors](https://github.com/codedtogether/web/graphs/contributors), this website was made possible.
```
Comment (abcdan) [inline README.md]: Again, this file shouldn't be needed as there is already a page easy to find on Github with all the contributors. I'm planning on adding an embed type thing with all contributors.

File: README.md (hunk 2 above)
Comment (SpyHoodle) [reply README.md]: Trying to make it sound more friendly to newcomers.

File: README.md (hunk 3 above)
Comment (SpyHoodle) [reply README.md]: Ok. Sure.

File: README.md (hunk 2 above)
Comment (SpyHoodle) [reply README.md]: I want to keep it though. Tell me if you really want it removed.

**Comment by SpyHoodle** [comment]
Done!

File: README.md (hunk 2 above)
Comment (abcdan) [reply README.md]: I don't think it's needed, as we can have a section on our website in the future discussing how it works and that the project is open source and such. @A-da-m @taneltm any opinions?

File: README.md (hunk 1 above)
Comment (SpyHoodle) [reply README.md]: This is one of the first things that you see. Having main info here is important, including links and stuff. It's best to mention it in here if people are going to 'READ' it.

File: README.md (hunk 2 above)
Comment (SpyHoodle) [reply README.md]: We are talking about *that* line you mentioned right? Because being kind is good on here.

File: README.md (hunk 1 above)
Comment (SpyHoodle) [reply README.md]: It directs you to this file if it wasn’t obvious.

**Comment by SpyHoodle** [comment]
README needs to have all the main info in one place. That’s why I added more stuff on there and made it friendly to someone who sees it.

**Comment by abcdan** [comment]
It doesn't _need_ to have all main info, we don't need things like "Yay" in it

**Comment by anishanne** [comment]
But it makes it seem more friendly and nice. However, I agree with @abcdan that we might be pushing the friendlyness a bit far.

**Comment by SpyHoodle** [comment]
@abcdan I've made a few changes.

File: README.md [hunk 4]
```diff
@@ -3,17 +3,19 @@ The [website](https://together.codes) for the code::together [Discord Server](ht
 Published under the [MIT Licence](https://github.com/codedtogether/web/blob/master/LICENSE).
 
 # Setup
-Want to see what we've done so far? Simply follow the steps.
```
Comment (abcdan) [inline README.md]: Thanks

File: README.md [hunk 5]
```diff
@@ -3,17 +3,19 @@ The [website](https://together.codes) for the code::together [Discord Server](ht
...
-1. Go to the `root` folder of the project, so where you are probably right now :)
+Follow these steps to view what we have done so far in your browser.
+1. Go to the `root` folder of the project, so where you are probably right now
 2. Run `npm install`
 3. Run `npm run dev & open http://localhost:3000`
 4. Navigate to `localhost:3000` in your browser
 
 
 # Like to contribute?
-Yay! We'd love your help!<br>
```
Comment (abcdan) [inline README.md]: Thanks

**Pull request merged**